container-aws-yaml/Pulumi.yaml

```diff
@@ -0,0 +1,72 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: yaml
+template:
+  description: A containerized service on AWS Fargate
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+    containerPort:
+      description: The port to expose on the container
+      default: 80
```

Comment:
When I run `pulumi new` and then `pulumi up` with this, I get:

```
➜  container-aws-yaml pulumi up
Previewing update (dev)

View Live: https://app.pulumi.com/cnunciato/container-aws-yaml/dev/previews/9f7ba2ab-1acc-4c84-bcf8-88f59cb085d7

error: failed to discover plugin requirements: Pulumi.yaml:7,18-20: default must be a string; default must be a string, and 2 other diagnostic(s)
```

Changing the values to `"80"`, `"512"`, and `"128"` get me past it, though. Is this expected? If not, I can file for it.